{{cookiecutter.project_slug}}/compose/production/django/entrypoint

```diff
@@ -4,12 +4,6 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
-
-{% if cookiecutter.use_celery == 'y' %}
-# N.B. If only .env files supported variable expansion...
-export CELERY_BROKER_URL="${REDIS_URL}"
-{% endif %}
```

Comment:
:tada:

Reply:
Adding a default value resolves this one to